packages/commcare/src/Adaptor.js

```diff
@@ -77,23 +78,62 @@ export function get(path, params = {}, callback = s => s) {
       params
     );
 
+    let { offset = 0, limit = 1000 } = resolvedParams;
```

Comment:
We should not default the limit here because it makes pagination weird (we only add the default on the second request, the first request will use the system default).

Reply:
ah, actually, we basically need to delete this line entirely